publish/filters.py

```diff
@@ -0,0 +1,11 @@
+from django_filters import rest_framework as filters
+
+from .models import Asset
+
+
+class AssetFilter(filters.FilterSet):
+    path_prefix = filters.CharFilter(lookup_expr='startswith', field_name='path')
+
+    class Meta:
+        model = Asset
+        fields = ['path', 'path_prefix']
```

Comment:
Do you still want the exact `path` matching filter? I'm fine if there's a reason to have it.

Reply:
I left it in there because I thought it might be useful, but I'm not sure if there's a real reason to keep it, so I can remove it.

Reply:
58fdcbb